src/statement.d

```diff
@@ -1324,38 +1329,38 @@ extern (C++) class CompoundStatement : Statement
         return statements;
     }
 
-    override final ReturnStatement isReturnStatement()
+    override final inout(ReturnStatement) isReturnStatement() inout nothrow pure
```

Comment:
How is it outside the scope of this? The function does not follow the convention used everywhere else for dynamic casting.